app/src/main/java/com/github/featuredetectandroid/utils/SelectFeatureDetector.kt

```diff
@@ -0,0 +1,15 @@
+package com.github.featuredetectandroid.utils
+
+import android.content.Context
+import com.github.featuredetectlib.learned.SuperPoint
+
+// TODO: add other FeatureDetector's after their implementation
+fun selectFeatureDetector(
+    context: Context,
+    algorithmName: String,
+    width: Int,
+    height: Int
+) = when (algorithmName) {
+    "SuperPoint" -> SuperPoint(context, width, height)
```

Comment:
We need to somehow tie `enum`s, names and classes in one place. Maybe create a class in which the `enum` will be nested and which will have methods like `enumToName` and `nameToClass`.

It's hard to maintain when these convertions are scattered all over the codebase.

Reply:
possible solution is to store in enum value string with title and clazz with type, or builder lambda

Reply:
fixed